packages/zmarkdown/index.js

```diff
@@ -43,6 +45,24 @@ const rebberConfig = require('./rebber-config')
 
 const fromFile = (filepath) => toVFile.readSync(filepath)
 
+const wrappers = {
+  iframe: [
+    new wrapperModule.Wrapper('div', ['video-container'],
```

Comment:
I think you could make it simpler: here you'd call `createWrapper('div', ['video-container'], …`, on l.103 you'd do:

```js
        nodeName => wrappers[nodeName].forEach(wrapper => visit(tree, nodeName, wrapper))
```

Reply:
don't understand, where does `createWrapper` comes from?